docs/classic-ui/templates-global-variables.md

```diff
@@ -0,0 +1,58 @@
+---
+html_meta:
+  "description": "Convenience global Plone variables in templates"
+  "property=og:description": "Convenience global Plone variables in templates"
+  "property=og:title": "Global variables in templates"
+  "keywords": "Plone, Classic UI, global variables, templates"
+---
+
+# Global variables in templates
+
+For convenience Plone defines a couple of global variables often used in templates.
+
+- {ref}`backend-global-utils-portal-state-label`
+- {ref}`backend-global-utils-context-state-label`
+- {ref}`backend-global-utils-plone-view-label`
+- {ref}`icons <classic-ui-icons-icon-resolver-label>`
+- `lang`
+- {ref}`classic-ui-templates-global-variables-portal-url-label`
+- {ref}`classic-ui-templates-global-variables-check-permissions`
+- `ajax_include_head`
+- `isAnon`
```

Comment:
Is there content for the above three items missing a link at this time?

Reply:
again these are variables defined in the main template and therefor globallay available in tempaltes, that's the main information. As soon as we have some chapters below, we can link them to detail infos, but they are useful even without already.

Reply:
Got it. This is good for now.